### PR TITLE
Update UI to match theme

### PR DESCRIPTION
**Pull request by tllshan**
Todos:
- [x] pie chart: 'no' label still shows with 0% no
- [x] resize window doesn't grow all the way vertically
- [X] empty diet cards visible
- [X] inconsistent margins
- [x] dietary panel cant be seen fully
- [x] command box font too light
- [x] pie doesn't match style
- [x] menu bar doesn't match style
- [x] Change piechart label to absolute numbers
- [x] Resize piechart (?)
- [x] Update DG diagram
- [x] Remove guest/vendor card select
- [x] Bigger response box

image

### CODE REVIEW

File: src/main/resources/view/MainWindow.fxml [hunk 1]
```diff
@@ -35,47 +35,39 @@
...
+            <StackPane fx:id="statisticsPlaceholder" prefWidth="200.0" maxHeight="100.0" styleClass="card"/>
+            <StackPane fx:id="rsvpPlaceholder" prefHeight="100.0" prefWidth="200.0" maxHeight="100.0" styleClass="card" />
+            <StackPane fx:id="dietaryPlaceholder" prefHeight="200.0" prefWidth="200.0" styleClass="card" />
           </VBox>
-          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS">
-            <padding>
-              <Insets bottom="10" left="10" right="10" top="10" />
-            </padding>
+          <VBox fx:id="guestList" prefWidth="350" styleClass="stack-pane" VBox.vgrow="ALWAYS" HBox.hgrow="ALWAYS">
             <StackPane fx:id="guestListPanelPlaceholder" VBox.vgrow="ALWAYS" />
```
Comment (p-xp) [inline src/main/resources/view/MainWindow.fxml:51]: The spacing on top of guestlist / vendorlist is inconsistent with the spacing between elements used throughout the rest of the app. It is narrower and thus seems slightly cramped. Would it be possible to make the spacing the same as every other place (e.g. the spacing between total count and rsvp status panel, and rsvp status and dietary panel)

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -35,47 +35,39 @@
...
+                <Insets bottom="10" left="10" right="10" top="10" />
+              </padding>
+            </StackPane>
           </VBox>
-
-
-
         </HBox>
 
         <StackPane fx:id="statusbarPlaceholder" prefWidth="405.0" VBox.vgrow="NEVER" />
```
Comment (p-xp) [inline src/main/resources/view/MainWindow.fxml:73]: Leaving the comment here as I'm not sure what's causing the issue. The application doesn't scale with size.
image

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -35,47 +35,39 @@
...
         </MenuBar>
 
-        <HBox prefWidth="400" style="-fx-background-color: #edede9">
-          <VBox styleClass="stack-pane">
-            <AnchorPane styleClass="split-pane">
-              <ImageView fx:id="logoImage" fitWidth="100.0" pickOnBounds="true" preserveRatio="true" AnchorPane.bottomAnchor="10.0" AnchorPane.leftAnchor="50.0" AnchorPane.rightAnchor="50.0" AnchorPane.topAnchor="10.0" />
-            </AnchorPane>
-            <StackPane fx:id="piechartPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
-            <StackPane fx:id="statisticsPlaceholder" prefHeight="400" prefWidth="200.0" styleClass="split-pane" />
+        <HBox HBox.hgrow="ALWAYS" prefWidth="400"  style="-fx-background-color: #f1e5e5">
```
Comment (p-xp) [inline src/main/resources/view/MainWindow.fxml:38]: Leaving this comment here as I can't comment on unedited portions.

The current menu bar design seems quite incongruous with the rest of the application, in terms of: font size, background colour (not sure if I'm seeing correctly but it seems to have a slight gradient), width.

**Review by p-xp — CHANGES_REQUESTED** [review]
Overall, really nice work on the design, and the app looks good 😄 Just some very minor nits to address.

**Review by p-xp — APPROVED** [review]
The dietary requirements panel looks a little squashed right now, would be quite difficult to view once we have more. Is it possible to remove the guest and vendor count box so that we have more vertical space for DR panel?
The guest and vendor count can be included at the top of the respective lists -- e.g. Guest(7), Vendor(3)

If this cannot be implemented in time for the demo, I am happy to merge as is.

**Comment by keaganpzh** [comment]
> The dietary requirements panel looks a little squashed right now, would be quite difficult to view once we have more. Is it possible to remove the guest and vendor count box so that we have more vertical space for DR panel? The guest and vendor count can be included at the top of the respective lists -- e.g. Guest(7), Vendor(3)
> 
> If this cannot be implemented in time for the demo, I am happy to merge as is.

In the interest of time, I can take over these tasks. For now, let's merge this PR.

**Pull request merged**